pkg/collector/python/datadog_agent.go

```diff
@@ -10,6 +10,7 @@ package python
 import (
 	"unsafe"
 
+	"github.com/DataDog/datadog-agent/pkg/trace/obfuscate"
```

Comment:
I forgot to ask: is the `obfuscate` works on windows ?

Reply:
I guess? The trace agent runs on windows and it already runs all sql queries that come through trace data, so it must work to some extent already.

Reply:
Confirmed that none of the `!windows` build flags in `pkg/trace` have an impact on the obfuscation code, so it should work the same on windows from a build perspective.